vars/cloudFoundryDeploy.groovy

```diff
@@ -135,11 +138,7 @@ def deployCfNative (config) {
             export HOME=${config.dockerWorkspace}
             cf login -u \"${username}\" -p '${password}' -a ${config.cloudFoundry.apiEndpoint} -o \"${config.cloudFoundry.org}\" -s \"${config.cloudFoundry.space}\"
             cf plugins
-            cf ${deployCommand} ${config.cloudFoundry.appName?"\"${config.cloudFoundry.appName}\"":''} -f \"${config.cloudFoundry.manifest}\" ${config.smokeTest}"""
-        def retVal = sh script: "cf app \"${config.cloudFoundry.appName}-old\"", returnStatus: true
-        if (retVal == 0) {
-            sh "cf delete \"${config.cloudFoundry.appName}-old\" -r -f"
-        }
+            cf ${deployCommand} ${config.cloudFoundry.appName?"'${config.cloudFoundry.appName}'":''} ${config.deployType == 'blue-green'?'--delete-old-apps':''} -f '${config.cloudFoundry.manifest}' ${config.smokeTest}"""
```

Comment:
[minor] I know the code is basically the same like before. but with the Elvis operator it would be much better to read. why not `cf ${deployCommand} ${config.cloudFoundry.appName ?: ''} ...`